test/elixir_console_web/features/console_test.exs

```diff
@@ -41,4 +41,17 @@ defmodule ElixirConsoleWeb.ConsoleTest do
       |> assert
     end)
   end
+
+  feature "visitor can shortcut used expressions", %{session: session} do
+    session
+    |> visit("/")
+    |> fill_in(css("#commandInput"), with: "a = 1 + 2")
```

Comment:
You can use the module attribute `@command_input` defined in this file instead of `css("#commandInput")`

Reply:
You are right!